### PR TITLE
Update github URLs Metaswitch->projectcalico

### PR DESCRIPTION
**Pull request by TrimBiggs**
Fix powerstrip version v0.4.8 refs to v0.4.9

### CODE REVIEW

File: docs/getting-started/VagrantUbuntu.md [hunk 1]
```diff
@@ -83,8 +83,8 @@ whether you are running with libnetwork, Powerstrip or the default Docker networ
 
 [libnetwork]: https://github.com/docker/libnetwork
 [experimental-channel]: https://github.com/docker/docker/tree/master/experimental
-[calico-ubuntu-vagrant]: https://github.com/Metaswitch/calico-ubuntu-vagrant-example
+[calico-ubuntu-vagrant]: https://github.com/projectcalico/calico-ubuntu-vagrant-example
```
Comment (robbrockbank) [inline docs/getting-started/VagrantUbuntu.md]: Not sure we reference this anymore.

File: docs/getting-started/powerstrip/EnvironmentSetup.md [hunk 2]
```diff
@@ -70,10 +70,10 @@ If you prefer not to do this you can still run the demo but remember to run
 ### Getting calicoctl Binary
 
 Get the calicoctl binary onto each host.  Don't use the latest 
-[release](https://github.com/Metaswitch/calico-docker/releases/) from github.
-Version v0.4.8 is required.
+[release](https://github.com/projectcalico/calico-docker/releases/) from github.
+Version v0.4.9 is required.
 
-	wget https://github.com/Metaswitch/calico-docker/releases/download/v0.4.8/calicoctl
+	wget https://github.com/projectcalico/calico-docker/releases/download/v0.4.9/calicoctl
```
Comment (robbrockbank) [inline docs/getting-started/powerstrip/EnvironmentSetup.md]: Did we actually test these against 0.4.9 ... I'm just surprised we didn't pick this up as part of the docs rework and testing.

Comment (TrimBiggs) [inline docs/getting-started/powerstrip/EnvironmentSetup.md]: Yes, I tested with 0.4.9, I updated the Vagrantfile and cloud-config when I was testing them during the docs update.

File: docs/getting-started/VagrantUbuntu.md (hunk 1 above)
Comment (TrimBiggs) [inline docs/getting-started/VagrantUbuntu.md]: Don't think so.  I'll get rid of it.

**Comment by robbrockbank** [comment]
lgtm

**Pull request merged**